docs/gestor_tareas.md

```diff
@@ -0,0 +1,62 @@
+# Gestor de Tareas
+La automatización de tareas es fundamental para optimizar los flujos de trabajo en el desarrollo de software. Este documento compara las herramientas más relevantes para seleccionar la más adecuada para proyectos en Java.
+
+## Criterios de selección
+Para elegir el gestor de tareas para nuestro proyecto en Java, hemos considerado los siguientes criterios:
+
+- *Comunidad*: Accesibilidad y disponibilidad de recursos de soporte, como foros oficiales, listas de correo, canales de comunicación como Slack o GitHub, y frecuencia de actualizaciones en la documentación oficial.
+- *Flexibilidad*:  Capacidad para adaptar y personalizar tareas a las necesidades específicas del proyecto. Se mide con número de niveles de personalización que ofrece cada herramienta, como la creación de nuevas tareas, modificación de procesos y reutilización de configuraciones.
```

Comment:
De veras, Evan, no pongáis esto. Ni es medible, ni es objetivo, ni es nada. Vamos a ver, ¿por qué has escogido gradle también como gestor de tareas? ¿Porque es flexible y tiene comunidad o por otra razón?

Reply:
Resuelto en https://github.com/evaanngiil/WishfulGiving/pull/19/commits/23940917c5d6b76c892b91b9a1933bb69ba40c9d